test/edns0.uts

```diff
@@ -60,10 +60,32 @@ raw(tlv) == b'\x00\x05\x00\x04\x00\x11"3'
 tlv = EDNS0TLV(optcode=2, optdata="")
 raw(tlv) == b'\x00\x02\x00\x00'
 
+= Util - setup retry_test()
+
+def retry_test(func):
```

Comment:
It could be nice to make `retry_test()` directly available from UTScapy.

Reply:
Done